internal/fuzzbinder/fuzzscraper.go

```diff
@@ -102,9 +100,10 @@ type Crash struct {
 // FuzzParameters contains the fuzzing parameters
 // used in OSS-Fuzz project config.
 type FuzzParameters struct {
-	ProjectName string
-	FuzzEngine  string
-	Sanitizer   string
+	ProjectName     string
+	ProjectHomepage string
```

Comment:
What is this? Can you add some documentation about it?

Reply:
I added a documentation for all of them and replaced the `ProjectHomepage` by the `ProjectGitRepo`.